src/ostree/ot-main.c

```diff
@@ -362,6 +363,12 @@ ostree_admin_option_context_parse (GOptionContext *context,
       exit (EXIT_SUCCESS);
     }
 
+  if ((flags & OSTREE_ADMIN_BUILTIN_FLAG_UNLOCKED) == 0)
+    {
+      if (!ot_admin_sysroot_lock (sysroot, error))
```

Comment:
As I had to look this up, I suggest a comment here to clarify the lock is implicitly released when the `OstreeSysroot` instance is finalized.